### ISSUE TITLE
Tabbing through product slider element highlights invisible products

### ISSUE THREAD

**Issue by makuhlmann** [shopware/platform#8680]
### Shopware Version

6.6.10.3

### Affected area / extension

Platform(Default)

### Actual behaviour

When tabbing through a product slider or a cross-selling slider that contains more items than there is space to display, tabbing through it does not work properly. When arriving at the element with tab, it first cycles through some products, but without visually highlighting them. Eventually the highlight will arrive at the visible products and cycle through them, however this could be confusing to people who use accessibility tools or screen readers.

Video demonstrating the issue - Observe the URL at the bottom left as I tab through the items:

https://github.com/user-attachments/assets/7c3a5638-0e73-4968-8026-6064a46c745f

### Expected behaviour

When arriving at the element using keyboard navigation, the items would be selected in order

### How to reproduce

- Create a shop, add 5 distinct products
- Create a new layout which contains a product slider, assign those 5 products
- Assign the layout to a landing page
- Open the landing page and try to navigate through the elements

**Comment by Phil23** [comment]
@makuhlmann Are you using the accessibility feature flag? There are a lot of improvements regarding accessibility, including keyboard navigation, that will ship with Shopware 6.7, but are also available in 6.6.x behind a feature flag.

**Comment by Phil23** [comment]
Closing this one, as it was fixed with 6.7 or using the accessibility feature flag.
